components/search/list-renderer/workflow.js

```diff
@@ -0,0 +1,41 @@
+// node_modules
+import PropTypes from "prop-types";
+// components/search/list-renderer
+import AlternateAccessions from "../../alternate-accessions";
+import {
+  SearchListItemContent,
+  SearchListItemMain,
+  SearchListItemMeta,
+  SearchListItemQuality,
+  SearchListItemTitle,
+  SearchListItemType,
+  SearchListItemUniqueId,
+} from "./search-list-item";
+
+export default function Workflow({ item: workflow }) {
+  return (
+    <SearchListItemContent>
+      <SearchListItemMain>
+        <SearchListItemUniqueId>
+          <SearchListItemType item={workflow} />
+          {workflow.accession}
+        </SearchListItemUniqueId>
+        <SearchListItemTitle>{workflow.name}</SearchListItemTitle>
+        <SearchListItemMeta>
+          <div key="lab">{workflow.lab.title}</div>
+          {workflow.alternate_accessions?.length > 0 && (
```

Comment:
You don’t have to change this, but for next time, `AlternateAccessions` already checks if alternate accessions exist or not, so you can just call it here instead of checking first.